src/nvim/eval.lua

```diff
@@ -118,6 +129,7 @@ M.funcs = {
     params = { { 'expr', 'number' }, { 'expr', 'number' } },
     returns = 'integer',
     signature = 'and({expr}, {expr})',
+    lua_alts = { '|bit.band()|' }
```

Comment:
could we use the existing `@see` handling?
```suggestion
    see = { '|bit.band()|' }
```